sources/PTYSession.m

```diff
@@ -1175,6 +1176,9 @@ + (void)finishInitializingArrangementOriginatedSession:(PTYSession *)aSession
         if (arrangement[SESSION_ARRANGEMENT_CURSOR_GUIDE]) {
             aSession.textview.highlightCursorLine = [arrangement[SESSION_ARRANGEMENT_CURSOR_GUIDE] boolValue];
         }
+        if (arrangement[SESSION_ARRANGEMENT_VCURSOR_GUIDE]) {
+            aSession.textview.highlightCursorLine = [arrangement[SESSION_ARRANGEMENT_VCURSOR_GUIDE] boolValue];
```

Comment:
This should assign to `highlightCursorCol` (which of course should be renamed to `highlightCursorColumn` 😊)

Reply:
Fixed.  Good catch!